packages/app/src/components-styled/time-series-chart/components/tooltip/tooltip-series-list.tsx

```diff
@@ -18,8 +18,10 @@ import { colors } from '~/style/theme';
 
 export function TooltipSeriesList<T extends TimestampedValue>({
   data: tooltipData,
+  columns,
```

Comment:
nit; you could define the default over here instead of on line 86
```suggestion
  columns = 1,
```